lib/model/model.go

```diff
@@ -1278,11 +1285,6 @@ func sendIndexTo(minSequence int64, conn protocol.Connection, folder string, fs
 			maxSequence = f.Sequence
 		}
 
-		if ignores.Match(f.Name).IsIgnored() || symlinkInvalid(folder, f) {
-			l.Debugln("not sending update for ignored/unsupported symlink", f)
-			return true
```

Comment:
I'm not super sure why we had this filtering here, but I'm fairly confident we shouldn't do it. If it does any difference at all it would screw up delta indexes as things with low sequence numbers would be mysteriously missing and never get sent.

However I don't see what we would ever have ignored files in our "have" list (unless they were not ignored previously, and then they should been sent then so we are fine) nor invalid symlinks (they are invalid so we can't have them locally).